cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -48,6 +48,10 @@ spec:
       # this needs to be set to false. For example, if using the local storage provisioner
       # this should be false.
       portable: true
+      # Certain storage class in the Cloud are slow
+      # Rook can configure the OSD running on PVC to accommodate that by tuning some of the Ceph internal
+      # Currently, "gp2" has been identified as such
```

Comment:
Is there a link we can provide or other brief info about how the user can know if this is the case?